Protobuf-cc.podspec

```diff
@@ -0,0 +1,107 @@
+Pod::Spec.new do |s|
+  s.name     = 'Protobuf-cc'
+  s.version  = '3.9.0-rc1'
+  s.summary  = 'Protocol Buffers v3 runtime library for C++.'
+  s.homepage = 'https://github.com/google/protobuf'
+  s.license  = '3-Clause BSD License'
+  s.authors  = { 'The Protocol Buffers contributors' => 'protobuf@googlegroups.com' }
+  s.cocoapods_version = '>= 1.0'
+
+  s.source = { :git => 'https://github.com/google/protobuf.git',
+               :tag => "v#{s.version}" }
+
+  s.source_files = 'src/google/protobuf/*.{h,cc,inc}',
+                   'src/google/protobuf/stubs/*.{h,cc}',
+                   'src/google/protobuf/io/*.{h,cc}',
+                   'src/google/protobuf/util/*.{h,cc}',
+                   'src/google/protobuf/util/internal/*.{h,cc}'
+
+  # Excluding all the tests in the directories above
+  s.exclude_files = 'src/google/protobuf/any_test.cc',
```

Comment:
Since `excludes_files` can also use globs, I wonder if it would be safer to do something like:

```
s.exclude_files = 'src/google/**/*_test.{h,cc,inc}',
                  'src/google/**/*_unittest.{h,cc}',
                  'src/google/protobuf/test_util*.{h,cc}',
```

That way it is less likely to accidentally include something that gets added in the future.

Reply:
Yes, this seems safer than explicitly specifying `exclude_files`, but unfortunately it doesn't work.

Compiling src/google/protobuf/map_lite_test_util.h fails as it depends on codegen for a test proto.
`'google/protobuf/map_lite_unittest.pb.h' file not found`

Reply:
Just add that to the exclusion:

```
s.exclude_files = 'src/google/**/*_test.{h,cc,inc}',
                  'src/google/**/*_unittest.{h,cc}',
                  'src/google/protobuf/test_util*.{h,cc}',
                  'src/google/protobuf/map_lite_test_util.{h,cc}'
```

Reply:
You might need to do the same thing for `map_test_util*.{h,cc}`

Reply:
Thanks! fixed.